blackcellmagic.py

```diff
@@ -38,7 +38,7 @@ def black(self, line, cell):
         line_length = args.line_length
         if cell:
             self.shell.set_next_input(
-                format_str(src_contents=cell, line_length=line_length),
+                format_str(src_contents=cell, line_length=line_length)[:-1],
```

Comment:
This is a perfectly valid ask. Lets just be cautious here and check that the last char is indeed `\n` before removing it so that the package doesn't break even if black decides to remove the empty line at the file sometime in future.

Reply:
Good idea. New commit is tested.